rskj-core/src/test/java/co/rsk/peg/constants/BridgeConstantsTest.java

```diff
@@ -1,11 +1,16 @@
-package co.rsk.config;
+package co.rsk.peg.constants;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 import co.rsk.bitcoinj.core.Coin;
 import java.util.stream.Stream;
+
+import co.rsk.peg.constants.BridgeConstants;
+import co.rsk.peg.constants.BridgeMainNetConstants;
+import co.rsk.peg.constants.BridgeRegTestConstants;
+import co.rsk.peg.constants.BridgeTestNetConstants;
```

Comment:
Is this required?

Reply:
I just optimized the imports in a new commit.